gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dcombobox_ttt2.lua

```diff
@@ -37,6 +37,8 @@ function PANEL:Init()
 	self.choices = {}
 	self.titleIndices = {}
 	self.valueIndices = {}
+
+	self.isValueTypeString = nil
```

Comment:
just name this valueType and write a string like "string", "number" "undefined" or something into it.

Reply:
Use these type definitions: https://wiki.facepunch.com/gmod/Global.type